src/views/input_field.rs

```diff
@@ -69,7 +59,7 @@ macro_rules! wrap_content_fun {
 
 impl InputField {
     pub const ENTER: KeyCombination = key!(enter);
-    pub const ALT_ENTER: KeyCombination = key!(alt-enter);
+    pub const ALT_ENTER: KeyCombination = key!(alt - enter);
```

Comment:
Sorry. It seems that the default `cargo fmt` has messed up all the `key!` macros. Do you have a project specific `rustfmt.toml` setting?

Reply:
As a potential workaround,`key!{alt-enter}` can be used in this case, as `rustfmt` ignores the arguments when the macro is called with `{}`.

Reply:
I don't have any rustfmt for this project, just don't use cargo fmt on it until I manage to make something correct (on most projects I must rely on nightly to get a reasonnable enough fmt).